inst/templates/contributing-template.txt

```diff
@@ -94,13 +94,14 @@ If you choose to contribute via GitHub,
 you may want to look at
 [How to Contribute to an Open Source Project on GitHub][how-contribute].
 In brief, we use [GitHub flow][github-flow] to manage changes:
-    1.  Create a new branch in your desktop copy of this repository for each significant change.
-    2.  Commit the change in that branch.
-    3.  Push that branch to your fork of this repository on GitHub.
-    4.  Submit a pull request from that branch to the [master repository][repo].
-    5.  If you receive feedback,
-        make changes on your desktop and push to your branch on GitHub:
-        the pull request will update automatically.
+
+1.  Create a new branch in your desktop copy of this repository for each significant change.
+2.  Commit the change in that branch.
+3.  Push that branch to your fork of this repository on GitHub.
+4.  Submit a pull request from that branch to the [master repository][repo].
```

Comment:
```suggestion
4.  Submit a pull request from that branch to the [upstream repository][repo].
```

This was something I noticed last time, but could not edit it because it wasn't part of the original PR.